assets/js/components/notifications/SwitchedToGA4Banner.js

```diff
@@ -67,14 +70,24 @@ export default function SwitchedToGA4Banner() {
 		return isValidPropertyID( propertyID );
 	} );
 
+	const eventCategory = `${ viewContext }_ga4-display-success-notification`;
+
+	const handleOnView = () => {
+		trackEvent( eventCategory, 'view_notification' );
+	};
```

Comment:
This function is used in an effect's dependency array in `BannerNotification` so it should be memoized with `useCallback`. This wouldn't result in duplicate events tracked due to the state that is there but would be an unnecessary recalculation of the effect on every render.

The other handler functions don't seem to be used in the same way so it's less critical but looking at other uses in the codebase we use `useCallback` for these as well, so let's do the same for consistency.

Reply:
Great catch, and thank you for the helpful context. Updated.